README.md

```diff
@@ -27,6 +27,18 @@ We currently support a very minimal set of environment variables to promote othe
 
 If you have any issues or questions, our [Community Forum](https://forum.sentry.io/c/on-premise) is at your service! Everytime you run the install script, it will generate a log file, `sentry_install_log-<ISO_TIMESTAMP>.txt` with the output. Sharing these logs would help people diagnose any issues you might be having.
 
+## Versioning
+
+We continously push the Docker image for each commit made into [Sentry](https://github.com/getsentry/sentry), and other services such as [Snuba](https://github.com/getsentry/snuba) or [Symbolicator](https://github.com/getsentry/symbolicator) to [our Docker Hub](https://hub.docker.com/u/getsentry) and tag the latest version on master as `:lastest`. This is also usually what we have on sentry.io and what the install script uses. You can use a custom Sentry image, such as a modified version that you have built on your own, or simply a specific commit (or pull request) by setting the `SENTRY_IMAGE` environment variable to that image name before running `./install.sh`:
```

Comment:
> or simply a specific commit (or pull request)

Maybe remove "(or pull request)" and add "hash" after commit? It's just commits in the end, people might be confused thinking there's a syntax for prs like `getsentry/sentry:$PR_ID`.

Also does `getsentry/sentry:<sha>` work? If so I'd just add another example.

Reply:
Good suggestions, applied them both <3